operations/research-and-development/product/release-process/boards-release.md

```diff
@@ -1,35 +1,34 @@
-# Focalboard Release Process
+# Boards Release Process
 
-This document outlines the Focalboard release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
+This document outlines the Boards release process for the Mattermost core team. Our release process draws from many best practice Agile, Scrum, and Software Development Lifecycle methodologies.
 
 ## Release Timeline
 
-Focalboard release schedule for July and August is outlined here: [https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl\_oJ-bAz1\_qpEiGhgF4HdDX1Uf4Q/edit\#gid=0](https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=0).
+Boards release schedule is outlined here: https://docs.google.com/spreadsheets/d/1nMrpSyR8PSEkHuQl_oJ-bAz1_qpEiGhgF4HdDX1Uf4Q/edit#gid=1279713913.
 
 The release that will be included in the next self-managed release pre-packaged version will generally be the latest Focalboard Personal Edition release from the prior month. The deadline to get the pre-packaged version to the self-managed release is by self-managed Code Freeze day. The pre-packaged version is tested a few days prior to self-managed Code Freeze day.
 
 ## Notes
 
-* **TBD:** Critical bugs are flagged and fixed before releasing, with quick dot releases completed as needed.
-* **TBD:** Incident Collaboration Playbook is used to run releases.
+* Critical bugs are flagged and fixed before releasing, with quick dot releases completed as needed.
```

Comment:
@amyblais Can potentially consider adding this bullet below line 13:
`Documentation on filing a bug: https://handbook.mattermost.com/contributors/contributors/ways-to-contribute#report-a-boards-bug`